config.ini

```diff
@@ -0,0 +1,2 @@
+[directories]
+battletech_home = C:\Program Files (x86)\Steam\steamapps\common\BATTLETECH
```

Comment:
Not sure about this. Battletech can be installed in infinite amount of places (especially if it's from GOG, Steam, different drives, folders)

My previous method of just looking from the root of the current script's location seemed fine, why change it?

Reply:
I wanted the ability to run the script directly from the repo, without copying it. The intention is that the user will configure this value, rather than copy the files into their Battletech install directory :)

Reply:
Ah I just now saw your Readme edit. Gotcha. Works for me!